Manual.md

```diff
@@ -132,6 +135,83 @@ the Void packages system.
 1. Required: Another package either within the repository or pending inclusion
    requires the package.
 
+<a id="cross_compilation"></a>
+### Cross compilation
+
+Void Linux places high emphasis on having packages being able to be
+cross-compiled.
+
+In the context of Void Linux cross compilation is compiling software from one
+CPU architecture to another, e.g. compile to aarch64 (64bits ARM) from x86_64
+(Intel/AMD 64bits) system.
+
+To help with cross compilation xbps-src employs a sizeable amount of complexity
+to make cross compiling as easy as running `./xbps-src -a <arch> pkg <pkgname>`
+for the user.
+
+<a id="cross_terminology"></a>
+#### Terminology
+
+The Void Linux cross compilation framework has specific terminology for various
+concepts that may or not match what other projects use.
+
+The following is a list of terminology used:
+
+- `Native`: A compilation that is from and to the same architecture. Compiling
+from x86_64 to x86_64 is native.
+
+- `Cross`: A compilation that is from one architecture to another. Compiling
+from x86_64 to aarch64 is cross.
+
+- `Host`: The machine in which we are compiling the program, taking the Cross
+example x86_64 would be the host.
+
+> NOTE: Most projects refer to this as 'build'
+
+- `Target`: The machine for which we are generating the program, taking the
+Cross example aarch64 would be the target.
+
+> NOTE: Most projects refer to this as 'host'
+
+<a id="cross_setup"></a>
+#### Cross setup
+
+Cross compiling requires some changes in the environment from variables being
+set to wrappers being made and packages being installed in specific locations.
+
+xbps-src sources common/cross-profiles/<arch>.sh which sets some important
+variables, among them `XBPS_TARGET_MACHINE` which is the name of the
+architecture according to xbps-src and `XBPS_CROSS_TRIPLET` which is the GNU
+triplet which can be acquired from running `gcc -dumpmachine`.
+
+xbps-src install packages from `makedepends` and `checkdepends` to
+`/usr/$XBPS_CROSS_TRIPLET`, that value is stored in the variable
+`XBPS_CROSS_BASE`.
```

Comment:
This is pretty ambiguous, `that value` could refer to both the path and make/checkdepends here.